src/python/pants/binaries/binary_tool.py

```diff
@@ -198,7 +201,7 @@ def hackily_snapshot(self, context):
     snapshot = context._scheduler.capture_snapshots((
       PathGlobsAndRoot(
         PathGlobs((script_relpath,)),
-        str(bootstrapdir),
+        native(str(bootstrapdir)),
```

Comment:
Hah... given that we call the pants engine backend the "native" backend, this was not confusing at all =)